### PR TITLE
Add mgmt identity

### PR DESCRIPTION
**Pull request by adusumillipraveen**
Adding KEDA to mgmt clusters as part of https://tools.hmcts.net/jira/browse/DTSPO-12434 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -7,3 +7,21 @@ resource "azurerm_user_assigned_identity" "managed_identity" {
...
   tags = var.common_tags
 }
+
+locals {
+  mgmt_env = ( var.env == "sandbox") ? "cftsbox-intsvc" : "cftptl-intsvc"
+}
+
+resource "azurerm_user_assigned_identity" "mgmt_managed_identity" {
+
+  count = (var.env == "sandbox" || var.env == "prod" ) ? 1 : 0
```
Comment (thomast1906) [inline main.tf:17]: Do we want it only for sbox & prod ?

Comment (adusumillipraveen) [reply main.tf:17]: i am just doing sbox ptl and prod ptl

Comment (adusumillipraveen) [reply main.tf:17]: its already on all other envs. A bit hacky  as general pipeline don't create infra in ptl envs. But thought keeping it here is better.

Comment (thomast1906) [reply main.tf:17]: does it not changed to reference similar to https://github.com/hmcts/cnp-keda-shared-infrastucture/pull/14/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR12 ?

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**